src/components/PhonicIcon/PhonicIcon.scss

```diff
@@ -0,0 +1,52 @@
+@import "../../pages/Index/index.scss";
+.PhonicIcon {
+    width: 20vw;
+    height: 20vw;
+    margin: 2em;
+    padding: 20px;
+    background-color: white;
+    border-radius: 2em;
+    box-shadow: 0em .2em #e6e6e6;
```

Comment:
have u used this color before bc u should also put it into a scss variable

Reply:
lol idk what that is so imma just delete that.